app/services/diagnostics/move_inspector.rb

```diff
@@ -107,63 +112,65 @@ def generate
+        else
+          @output << "(no person associated with move)\n"
+        end
 
-      @output << <<~PROFILEDETAILS
-
-        PROFILE
-        -------
-      PROFILEDETAILS
-
-      if @move.profile.present?
-
-        @output << "id:\t#{move.profile.id}\n"
-        @output << "created at:\t#{move.profile.created_at}\n"
-        @output << "updated at:\t#{move.profile.updated_at}\n"
-
-        @output << "\nASSESSMENT ANSWERS\n"
-        @output << "------------------\n"
-        if move.profile.assessment_answers.any?
-          move.profile.assessment_answers.each do |answer|
-            @output << "title:\t#{answer.title}\n"
-            @output << "key:\t#{answer.key}\n"
-            @output << "category:\t#{answer.category}\n"
-            @output << "comments:\t#{answer.comments}\n"
-            @output << "created at:\t#{answer.created_at}\n"
-            @output << "expires at:\t#{answer.expires_at}\n"
-            @output << "nomis_alert_type:\t#{answer.nomis_alert_type}\n"
-            @output << "nomis_alert_code:\t#{answer.nomis_alert_code}\n"
-            @output << "nomis_alert_type_description:\t#{answer.nomis_alert_type_description}\n"
-            @output << "nomis_alert_description:\t#{answer.nomis_alert_description}\n"
-            @output << "imported_from_nomis:\t#{answer.imported_from_nomis}\n"
-            @output << "---\n"
+        @output << <<~PROFILEDETAILS
+
+          PROFILE
+          -------
+        PROFILEDETAILS
+
+        if @move.profile.present?
+
+          @output << "id:\t#{move.profile.id}\n"
+          @output << "created at:\t#{move.profile.created_at}\n"
+          @output << "updated at:\t#{move.profile.updated_at}\n"
+
+          @output << "\nASSESSMENT ANSWERS\n"
+          @output << "------------------\n"
+          if move.profile.assessment_answers.any?
+            move.profile.assessment_answers.each do |answer|
+              @output << "title:\t#{answer.title}\n"
+              @output << "key:\t#{answer.key}\n"
+              @output << "category:\t#{answer.category}\n"
+              @output << "comments:\t#{answer.comments}\n"
+              @output << "created at:\t#{answer.created_at}\n"
+              @output << "expires at:\t#{answer.expires_at}\n"
+              @output << "nomis_alert_type:\t#{answer.nomis_alert_type}\n"
+              @output << "nomis_alert_code:\t#{answer.nomis_alert_code}\n"
+              @output << "nomis_alert_type_description:\t#{answer.nomis_alert_type_description}\n"
+              @output << "nomis_alert_description:\t#{answer.nomis_alert_description}\n"
+              @output << "imported_from_nomis:\t#{answer.imported_from_nomis}\n"
+              @output << "---\n"
+            end
+          else
+            @output << "(no assessment answers recorded)\n"
           end
         else
-          @output << "(no assessment answers recorded)\n"
+          @output << "(no profile associated with move)\n"
```

Comment:
This is a very long method - not the end of the world as it's all pretty simple stuff but perhaps potentially a candidate for splitting into smaller sections?

Reply:
yes.. fair enough - will save that for a rainy day